packages/api/src/controllers/usage.ts

```diff
@@ -32,6 +34,19 @@ app.post(
   async (req, res) => {
     let { fromTime, toTime } = req.query;
 
+    const authHeader = req.headers.authorization;
+    let token = "";
+    if (authHeader) {
+      token = authHeader.split(" ")[1];
+    }
```

Comment:
You can probably just use req.token
```suggestion
    const token = req.token;
```